src/main/java/core/basesyntax/MainApp.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class MainApp {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Random figure = new Random();
+        int num = figure.nextInt(5) + 1;
+
+        for (int i = 0; i < num; i++) {
+            Figure randomFigure = figureSupplier.getRandomFigure();
+            System.out.println(randomFigure.behaviour());
+        }

```

Comment:
you need to generate random number of random figures

Reply:
but it's generate random number of random figures, or I don't understand something. Please explain what should I do